osg_configure/configure_modules/bosco.py

```diff
@@ -290,9 +294,12 @@ def result():
                     self.log("stderr:\n%s" % stderr, level=logging.ERROR)
                     return False
 
-
             # Run bosco cluster to install the remote cluster
-            install_cmd = ["bosco_cluster", "-a", endpoint, batch]
+            install_cmd = ["bosco_cluster"]
+            patch_dir = self.options['patch_dir'].value
```

Comment:
```suggestion
            patch_dir = self.opt_val("patch_dir")
```
I've been trying to use `opt_val` instead of `options[...].value` because it has less potential for error (i.e. forgetting the `.value`) so let's be consistent.